analyzer/restapi-plugin/src/test/java/eu/fasten/analyzer/restapiplugin/mvn/api/PackageApiTest.java

```diff
@@ -46,7 +46,6 @@ void setUp() {
         service = new PackageApi();
         kbDao = Mockito.mock(MetadataDao.class);
         KnowledgeBaseConnector.kbDao = kbDao;
-        KnowledgeBaseConnector.forge = "mvn";
```

Comment:
Hi @gdrosos This change is breaking your tests :-) You should add it back like:
`KnowledgeBaseConnector.forge = Constants.mvnForge;`

Reply:
No problem, almost feels like pair programming :-)